proarc-common/src/main/java/cz/cas/lib/proarc/common/imports/TiffImporter.java

```diff
@@ -67,10 +67,10 @@
  *
  * @author Jan Pokorsky
  */
-public final class TiffImporter {
+public class TiffImporter implements ImageImporter {
```

Comment:
Ano, takhle je to správně. To byl komentář k původní verzi. Ještě tam zůstal pozůstatek
```
-    private static final Logger LOG = Logger.getLogger(TiffImporter.class.getName());
-    private final ImportBatchManager ibm;
+    protected static final Logger LOG = Logger.getLogger(TiffImporter.class.getName());
+    protected final ImportBatchManager ibm;
```
Nic z toho jsem nenašel použité v potomcích. Takže by to mělo být zpět private.